### ISSUE TITLE
Do you have support for adding dynamic WebExtension element nodes in Word Document?

### ISSUE THREAD

**Issue by rake93** [aspose-words/Aspose.Words-for-Java#52]
I have requirement where I need to add Web Extension properties to the document dynamically when it downloaded from application, below is the sample webextension xml that I need to add.

```
<we:webextension xmlns:we="http://schemas.microsoft.com/office/webextensions/webextension/2010/11" id="{52811C31-4593-43B8-A697-EB873422D156}">
    <we:reference id="af8fa5ba-4010-4bcc-9e03-a91ddadf6dd3" version="1.0.0.0" store="EXCatalog" storeType="EXCatalog"/>
    <we:alternateReferences/>
    <we:properties/>
    <we:bindings/>
    <we:snapshot xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships"/>
</we:webextension>
```
I have gone through your API Documentation and examples and did'nt find any thing related to adding Web Extension properties to the Document, but I found that we can add to the Excel Workbook from below link.
https://docs.aspose.com/display/cellsjava/Web+Extensions+-+Office+Add-ins#WebExtensions-OfficeAdd-ins-SampleCode

Would you please confirm that you have support for Word document as well?  If so, could you please provide me sample code or the steps to achieve this requirement.

**Comment by awaishafeez** [comment]
@rake93 ,

We have logged your requirement in our issue tracking system. Your ticket number is WORDSNET-18634. We will further look into the details of this requirement and will keep you updated on the status here.

**Comment by rake93** [comment]
@awaishafeez - Any update on my request?

**Comment by awaishafeez** [comment]
@rake93,
Unfortunately, WORDSNET-18634 is not resolved yet. We have completed the initial analysis of this issue and will inform you once this issue will be resolved in future. We apologize for any inconvenience.

**Comment by AlexNosk** [comment]
@rake93 The feature was included into the 19.11 version of Aspose.Words. Please check [release notes ](https://docs.aspose.com/display/wordsjava/Aspose.Words+for+Java+19.11+Release+Notes) for more information.

**Comment by rake93** [comment]
@AlexNosk - Thanks for the update.
